internal/services/storage/resourceids.go

```diff
@@ -7,4 +7,5 @@ package storage
 //go:generate go run ../../tools/generator-resource-id/main.go -path=./ -name=StorageAccountDefaultBlob -id=/subscriptions/12345678-1234-9876-4563-123456789012/resourceGroups/resGroup1/providers/Microsoft.Storage/storageAccounts/storageAccount1/blobServices/default
 //go:generate go run ../../tools/generator-resource-id/main.go -path=./ -name=StorageQueueResourceManager -id=/subscriptions/12345678-1234-9876-4563-123456789012/resourceGroups/resGroup1/providers/Microsoft.Storage/storageAccounts/storageAccount1/queueServices/default/queues/queue1
 //go:generate go run ../../tools/generator-resource-id/main.go -path=./ -name=StorageShareResourceManager -id=/subscriptions/12345678-1234-9876-4563-123456789012/resourceGroups/resGroup1/providers/Microsoft.Storage/storageAccounts/storageAccount1/fileServices/fileService1/fileshares/share1
+//go:generate go run ../../tools/generator-resource-id/main.go -path=./ -name=LegacyStorageShareResourceManager -id=/subscriptions/12345678-1234-9876-4563-123456789012/resourceGroups/resGroup1/providers/Microsoft.Storage/storageAccounts/storageAccount1/fileServices/fileService1/shares/share1
```

Comment:
It looks like there's already a resource id for this and the api isn't accepting/returning the correct id so we can't transform between the incorrect to the correct based on these issues: https://github.com/Azure/azure-rest-api-specs/issues/25209, https://github.com/Azure/azure-rest-api-specs/issues/24568 . Rather than introducing a new resource id we would need the service team to fix this first.

Reply:
OK. I removed this feature from this PR. This feature can be introduced once this issue is resolved in the feature.